example_projects/transfer/nlb_observability_stack/transfer.tf

```diff
@@ -0,0 +1,236 @@
+resource "doublecloud_transfer_endpoint" "nlb-s3-s32ch-source" {
+  name       = var.transfer_source_name
+  project_id = var.project_id
+  settings {
+    object_storage_source {
+      provider {
+        bucket                = var.bucket_name
+        path_prefix           = var.bucket_prefix
+        aws_access_key_id     = var.aws_access_key_id
+        aws_secret_access_key = var.aws_access_key_secret
+        region                = var.region
+        endpoint              = var.endpoint
+        use_ssl               = true
+        verify_ssl_cert       = false
```

Comment:
Looks not production ready

Reply:
why ?

Reply:
cause in production you should verify SSL